.github/workflows/periodic.yml

```diff
@@ -161,6 +163,7 @@ jobs:
 
   ios-12-5-1-arm64-custom-ops:
     name: ios-12-5-1-arm64-custom-ops
+    if: false # see https://github.com/pytorch/pytorch/issues/82890
```

Comment:
Do comments work?

Reply:
as far as i can tell, this one seems fine, i added the ciflow/periodic label and they did get skipped